app/services/create_admin_service.rb

```diff
@@ -0,0 +1,9 @@
+class CreateAdminService
+  def call
+    user = User.find_or_create_by!(email: Rails.application.secrets.admin_email) do |user|
```

Comment:
why do we need to create admin users? I think user could be registered and then simply promoted by SuperAdmin to Admin or another role

Reply:
Now I have three types of users:
- admin -> superadmin
- user - will have less abilities than admin
- guest - do not exist. the person who visited website and can only read contents

Reply:
This service is used with db:seed to create admin

Reply:
`Rails.application.secrets.admin_password`, instead you could just have `password = "test"` in your seed file

Reply:
replaced with recommended solution